server/src/main/java/org/elasticsearch/action/admin/cluster/stats/ClusterStatsNodes.java

```diff
@@ -137,8 +136,8 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.endObject();
 
         builder.startArray(Fields.VERSIONS);
-        for (Version v : versions) {
-            builder.value(v.toString());
+        for (final var v : versions) {
```

Comment:
Don't think the `final` adds anything here...

Reply:
It does not hurt either :) but I can remove it